onyo/lib/commands.py

```diff
@@ -575,6 +575,89 @@ def onyo_get(inventory: Inventory,
     return results
 
 
+@raise_on_inventory_state
+def onyo_history(inventory: Inventory,
+                 path: Path,
+                 interactive: bool | None = None) -> None:
+    r"""Display the history of a path.
+
+    Only one ``path`` is accepted as that's ``git log --follow``'s limitation.
+    This is due to git's design, and is independent of the tool used to query it.
```

Comment:
Hm. TBH: I don't buy it. Of course I can build a tool that takes several paths to follow. Not different from showing all current branch tips, etc. 
I don't know it exists, but it absolutely could.

Reply:
I had a similar impulse, but then started reading and people seemed very convinced of the complexity of the issue.

I do feel comfortable keeping this limitation for now, as neither `git` nor `tig` support it.

That said, if you feel strongly about it, I can adjust this. I would take the approach of adding a script (perhaps to `.onyo/bin`) to run `tig`, as its error message for passing multiple arguments to `--follow` is not helpful. That may or may not be its own can of worms.

Reply:
I'm kinda fine with the limitation, even though I'd think it nicer to let the configured command error.
I'm not fine with that claim.
Obviously, I can say `git log --follow` on several paths, simply by having one call per path. To claim, that there can't be a tool that collects that information and show it in a useful way is a really bold claim, that I see no basis for.

Reply:
The only thing not trivial about it I ca see, is to figure out whether one can usefully draw a shared graph in 2D and/or how to break it up into a minimal number of such graphs. As opposed to treating the given paths separately (which is - again - trivial and obvious)

Reply:
Pushed.